### ISSUE TITLE
[Bug] Da Bomb Skill isn't being calculated correctly

### ISSUE THREAD

**Issue by Vvnguyen25** [LLAMARPG/game#53]
When using Da Bomb, my next attack should be increased by 90%. My item does 12 spark damage. So in theory it should do 22.8 damage. But looking at the Damage log it looks like the weapon did 15 spark damage and an additional 9 spark damage.

Skill of Da Bomb:
![image](https://github.com/LLAMARPG/game/assets/172426161/91de23ae-6c48-4a89-ad94-2635c75a4ad5)

Weapon Melee Attack used:
![image](https://github.com/LLAMARPG/game/assets/172426161/04d77a63-6dac-46e9-bdde-4f3b226d734b)

Combat Log:
![image](https://github.com/LLAMARPG/game/assets/172426161/fd500cc0-8e03-4465-b16d-6bf7554f88bd)

**Comment by Warren1001** [comment]
its worth noting that since monster resistances dont currently display, this may be because of high resistance

**Comment by Vvnguyen25** [comment]
> its worth noting that since monster resistances dont currently display, this may be because of high resistance. all monsters have some degree of resistance to mostly everything, with typically at least 1 resistance being particularly notable

Do you know if monsters can take bonus damage then too?
I used a spell twice and it should have added up to 60 but the monster took 81 burn damage at the end of turn:
![image](https://github.com/LLAMARPG/game/assets/172426161/97d4285d-7c42-4c20-8a16-87f587193449)

**Comment by Warren1001** [comment]
yes, there are glancing blows and critical strikes, doing 0.5x and 1.5x respectively. if the monster has resistance to burn, then the math would check out if you crit. this is something that needs to be made more obvious (when a crit or glance happen) as theres no way to tell other than doing the math
